### DISCUSSION TITLE
Ingress获取真实IP | Qinng

### DISCUSSION THREAD

**Discussion by qingwave** [qingwave/qingwave.github.io#63] [Announcements]
https://qingwave.github.io/ingress-real-ip/ 

 Coding, Reading and Daydreaming

**Comment by yuedingwangji** [comment]
能指定单个域名生效么

**Comment by qingwave** [comment]
> @yuedingwangji
> 能指定单个域名生效么

目前这个配置是全局生效的，还不支持单个ingress
